tools/olm-bundle/README.md

```diff
@@ -0,0 +1,32 @@
+# OLM bundle generator
+Bash script for generating an OLM bundle for OperatorHub from existing CRD, ClusterRole, Configmap, and Deployment files.
+The generator copies and renames these existing files from the `../../packaging/install/cluster-operator` directory.
+
+Then the generator creates a CSV file generating the following CSV fields automatically:
+- `metadata.name`
+- `metadata.annotations.containerImage`
+- `metadata.annotations.createdAt`
+- `spec.name`
+- `spec.version`
+- `spec.replaces`
+- `spec.install.spec.permissions`
+- `spec.install.spec.clusterPermissions`
+- `spec.install.spec.deployments`
```

Comment:
How does this work? I guess you do it with `yq`?

Reply:
Yes, these are copied over from the existing CRD/install files using `yq`

Do you think I should add more information about how it is done to the README?